packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -62,6 +63,30 @@ const propTypes = {
     ),
   ),
   /**
+   * An array of nav items to be displayed on the user profile/ settings menu/popup.
+   */
+  profileLinks: PropTypes.arrayOf(PropTypes.shape(
+    navElementShape)),
+  /**
+   * User name to be displayed in the profile in navigation.
+   */
+  userName: PropTypes.string.isRequired,
+  /**
+   * Avatar to be displayed in user profile in navigation.
+   */
+  avatar: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.element,
+  ]),
+  /**
+   * A unique id set to the profile popup button that will be referred in profile popup.
+   */
+  profileId: PropTypes.string.isRequired,
```

Comment:
Is there a reason this id needs to be exposed to the consumer? Could it just be something namespaced and unique inside this component?